pkg/curatedpackages/cptabwriter_test.go

```diff
@@ -0,0 +1,52 @@
+package curatedpackages
+
+import (
+	"bytes"
+	"fmt"
+	"testing"
+)
+
+func TestCPTabwriterDefaultParams(t *testing.T) {
+	buf := &bytes.Buffer{}
+	// params := &cpTabwriterDefaultParams
+	// params.flags |= tabwriter.Debug
+	w := newCPTabwriter(buf, nil)
+
+	baseBuf := &bytes.Buffer{}
+	baseline := newCPTabwriter(baseBuf, nil)
+	fmt.Fprint(baseline, "one\ta\t\ntwo\tb\t\nthree\tc\t\n")
+	baseline.Flush()
+
+	err := w.writeTable([][]string{{"one", "a"}, {"two", "b"}, {"three", "c"}})
+	if err != nil {
+		t.Fatalf("expected nil, got %v", err)
+	}
+	w.Flush()
+
+	if baseBuf.String() != buf.String() {
+		t.Fatalf("expected %q, got %q", baseBuf.String(), buf.String())
+	}
+}
+
+func TestCPTabwriterCustomPadChar(t *testing.T) {
+	buf := &bytes.Buffer{}
+	params := &cpTabwriterDefaultParams
+	params.padChar = '='
+	// params.flags |= tabwriter.Debug
```

Comment:
Is this needed here?

Reply:
This and the one above it, I left there as reminders for anyone trying to debug them. I can take them or leave them. Since I'll be adding test coverage, I guess I'll remove them.